packages/app/next.config.js

```diff
@@ -11,6 +11,15 @@ const withNextra = require('nextra')({
 module.exports = {
   experimental: {
     instrumentationHook: true,
+    // External packages to prevent bundling issues with Next.js 14
+    // https://github.com/open-telemetry/opentelemetry-js/issues/4297#issuecomment-2285070503
+    serverComponentsExternalPackages: [
+      '@opentelemetry/instrumentation',
+      '@opentelemetry/sdk-node',
+      '@opentelemetry/auto-instrumentations-node',
+      '@hyperdx/node-opentelemetry',
+      '@hyperdx/instrumentation-sentry-node',
+    ],
```

Comment:
Needed because of stricter bundling requirements in nextJS 14. This is the official workaround.